src/components/UI/Alert/DialogBox.vue

```diff
@@ -10,18 +10,21 @@
           :src="icon"
           class="place-self-center"
           :class="iconClass"
+          data-test="icon"
         ></inline-svg>
         <!-- title -->
         <p
           class="text-gray-800 dark:text-gray-200 text-xl font-bold mt-4"
           v-if="showTitle"
+          data-test="title"
```

Comment:
Did you check that multiple data-tests with the same name do not conflict in the main app?